cdap-cli/src/main/java/co/cask/cdap/cli/command/DeleteAdapterCommand.java

```diff
@@ -56,6 +56,6 @@ public String getPattern() {
 
   @Override
   public String getDescription() {
-    return "Deletes a " + ElementType.ADAPTER.getPrettyName();
+    return "Deletes an " + ElementType.ADAPTER.getPrettyName();
```

Comment:
How do you "an" is correct here? Perhaps have a ``getAnAndPrettyName()`` that returns the correct indefinite article for the name, with a separating space...